### PR TITLE
[HSEARCH] 5.10.2.Final release

### PR DESCRIPTION
**Pull request by yrodiere**
On staging: http://staging.in.relation.to/2018/06/25/hibernate-search-5-10-2-Final/

### CODE REVIEW

File: posts/Yoann/2018-06-25-hibernate-search-5-10-2-Final.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Second maintenance release for Hibernate Search 5.10
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published Hibernate Search version 5.10.2.Final,
+the second maintenance release of Hibernate Search.
```
Comment (gsmet) [inline posts/Yoann/2018-06-25-hibernate-search-5-10-2-Final.adoc]: ever? :)

File: posts/Yoann/2018-06-25-hibernate-search-5-10-2-Final.adoc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+We took this opportunity to backport the parts of the Query DSL that allow to set `minimumShouldMatch` constraints
+from the next version of Hibernate Search.
+You can now simply use code similar to this:
+
+```
+FullTextEntityManager fullTextEntityManager = Search.getFullTextEntityManager( entityManager );
+QueryBuilder queryBuilder = fullTextEntityManager.getSearchFactory()
+		.buildQueryBuilder().forEntity( MyEntity.class ).get();
+Query luceneQuery = queryBuilder.bool()
+		.minimumShouldMatchNumber(2) // This is new
```
Comment (gsmet) [inline posts/Yoann/2018-06-25-hibernate-search-5-10-2-Final.adoc]: missing spaces if you want to be consistent.

**Review by gsmet — COMMENTED** [review]
Added a few comments inline.

**Comment by yrodiere** [comment]
Fixed and merged, thanks!

**Pull request merged**